gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -774,7 +841,7 @@ int ROOT::Experimental::TWebWindowsManager::WaitFor(TWebWindow &win, WebWindowWa
       spent = elapsed.count() * 1e-3; // use ms precision
 
       if (timed && (spent > timelimit))
-         return 0;
+         return -3;
```

Comment:
Where is the meaning of `-3` documented?

Reply:
Please consider introducing an enum for user-facing error codes. In a subsequent PR :-)

Reply:
May be. It is still not very clear which interface here should be exposed to users.